bytecomp/simplif.ml

```diff
@@ -566,7 +566,7 @@ let rec emit_tail_infos is_tail lambda =
   | Lletrec (bindings, body) ->
       List.iter (fun (_, lam) -> emit_tail_infos false lam) bindings;
       emit_tail_infos is_tail body
-  | Lprim (Pidentity, [arg]) ->
+  | Lprim ((Pidentity | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
you are right, for this simplification, we don't need it (and it will still not change the code gen) I will update